package.json

```diff
@@ -30,7 +30,7 @@
     "extend": "^2.0.0",
     "form-data2": "^1.0.0",
     "form-fix-array": "^1.0.0",
-    "lodash": "^2.4.1",
+    "lodash": "^4.17.10",
```

Comment:
There is already a vulnerability reported by `npm audit` for this version (https://www.npmjs.com/advisories/1065).

Reply:
Why not update to 4.17.15, which was published 25 days ago?

Reply:
@settermjd updated to that version